migrations/20220523141538_recipe_transactions_table.js

```diff
@@ -1,26 +1,21 @@
 exports.up = async (knex) => {
-  // check extension is not installed
-  const [extInstalled] = await knex('pg_extension').select('*').where({ extname: 'uuid-ossp' })
-
-  if (!extInstalled) {
-    await knex.raw('CREATE EXTENSION "uuid-ossp"')
-  }
-
   const uuidGenerateV4 = () => knex.raw('uuid_generate_v4()')
   const now = () => knex.fn.now()
 
   await knex.schema.createTable('recipe_transactions', (def) => {
     def.uuid('id').defaultTo(uuidGenerateV4())
     def.integer('token_id').unsigned().notNullable()
     def.uuid('recipe_id').notNullable()
-    def.enu('status', ['submitted', 'minted'])
+    def.enu('type', ['Creation']).notNullable()
```

Comment:
not saying we fix this here but where this `type` column has been added we need to make sure it's populated on insert. What I am surprised by is that this hasn't further broken the tests from #15. You may want to investigate that

Reply:
this is because we stub database methods and tests have not been updated to expect new properties, also the api validation will be tested in integration tests along with database in/out. this is purely as a unit test to confirm that it calls database.

We could potentially use spy rather than stub i think using it might give us a benefit of actually using db client, but not sure

Reply:
to be fair I was looking at the recipe example

so the transactions have been updated:

```js
      id: 'TRASACTI-0000-2000-8000-000000000000',
      recipe_id: 'RECIPE00-0000-1000-8000-00000000000',
      token_id: 'token-id',
      status: 'InBlock',
      created_at: '2022-05-23T13:04:29.316Z',
```

Reply:
well it does not have type tho

Reply:
so is this updated to include `type`?